.travis.yml

```diff
@@ -0,0 +1,89 @@
+#=============================================================================
+# Travis CoreNeuron settings
+#=============================================================================
+
+#=============================================================================
+# Environment
+#=============================================================================
+# Use new Travis infrastructure (Docker can't sudo yet)
```

Comment:
Does this comment make sense, given that `suda: false` ?

Reply:
It seems that "sudo" key is deprecated in travis so I would remove it and that comment.